src/index.js

```diff
@@ -1,12 +1,14 @@
 // @flow
 import {registerPlugin} from '@playkit-js/playkit-js';
-import {Youbora} from './youbora';
+import {Plugin} from './youbora';
```

Comment:
```suggestion
import {Youbora as Plugin} from './youbora';
```
like you do in ima/ima-dai